config/user_sql_constants.js

```diff
@@ -0,0 +1,354 @@
+/*ALL SQL CONSTANTS FOR USER CONTROLLER */
+
+//create mentor, mentee and matchces table for testing 
+module.exports.create_mentee_table_sql = function()  {
+    sql = `    
+        CREATE TABLE IF NOT EXISTS Mentees (
+            user_id int NOT NULL UNIQUE,
+            first_name varchar(255) NOT NULL,
+            last_name varchar(255) NOT NULL,
+            email_address varchar(255) NOT NULL UNIQUE,
+            biography varchar(255),
+            zipcode varchar(5) NOT NULL,
+            date_of_birth DATE NOT NULL,
+            area_of_study varchar(255) NOT NULL,
+            skills varchar(255) NOT NULL,
+            profile_pic_URL varchar(255) NOT NULL,
+            hobbies varchar(255)
+        );`
+    return sql; 
+}
+
+module.exports.create_mentor_table_sql = function()  {
+    sql = `  
+    CREATE TABLE IF NOT EXISTS Mentors ( 
+        user_id int NOT NULL UNIQUE,
+        first_name varchar(255) NOT NULL,
+        last_name varchar(255) NOT NULL,
+        email_address varchar(255) NOT NULL UNIQUE,
+        biography varchar(255),
+        zipcode varchar(5) NOT NULL,
+        date_of_birth DATE NOT NULL,
+        occupation varchar(255) NOT NULL,
+        skills varchar(255) NOT NULL,
+        profile_pic_URL varchar(255) NOT NULL,
+        hobbies varchar(255)
+    );`
+    return sql; 
+}
+module.exports.create_password_table_sql = function()  {
+    sql = `  
+    CREATE TABLE IF NOT EXISTS Passwords ( 
+        user_id int NOT NULL UNIQUE,
+        email_address varchar(255) NOT NULL UNIQUE,
+        password varchar(255) 
+    );`
+    return sql; 
+}
+
+module.exports.create_matches_table_sql = function()  {
+    sql = `CREATE TABLE IF NOT EXISTS Matches (
+        match_id int NOT NULL UNIQUE,
+        mentor_id int NOT NULL,
+        mentee_id int NOT NULL
+    );`
+    return sql; 
+}
+
+module.exports.create_messages_table_sql = function()  {
+    sql = `CREATE TABLE IF NOT EXISTS Messages (
+        message_id int NOT NULL UNIQUE,
+        match_id int NOT NULL,
+        to_id int NOT NULL,
+        from_id int NOT NULL,
+        message_body varchar(255),
+        timestamp datetime NOT NULL
+    );`
+    return sql; 
+}
+
+//create new mentor
+module.exports.post_mentor_sql = function(user)  {
+    sql = `INSERT INTO Mentors VALUES ('${user.user_id}', '${user.first_name}', '${user.last_name}', '${user.email_address}', 
```

Comment:
Avoid injecting the string into the sql query directly. Use the `?` placeholder and pass the string via array to the `db.get` and `db.all` functions instead. See http://www.sqlitetutorial.net/sqlite-nodejs/query/.